src/components/user/tour/create-tour/guests.js

```diff
@@ -0,0 +1,204 @@
+import React, {Component} from "react";
+import Button from "@material-ui/core/Button";
+import {DataGrid} from "@material-ui/data-grid";
+import TextField from "@material-ui/core/TextField";
+import Checkbox from "@material-ui/core/Checkbox";
+import CheckBoxOutlineBlankIcon from '@material-ui/icons/CheckBoxOutlineBlank';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import {GUESTS_COLUMNS, GUESTS_ROWS} from "../utils/constants";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+
+export default class Guest extends Component {
```

Comment:
This component also should be rewritten with the usage of Formik. Will review this file one more time, after changing it.